src/chat/channel.py

```diff
@@ -51,13 +51,13 @@ def __init__(self, lobby, name, private=False, *args, **kwargs):
         BaseClass.__init__(self, lobby, *args, **kwargs)
 
         self.setupUi(self)
-        
+
         #Special HTML formatter used to layout the chat lines written by people
         self.lobby = lobby
         self.chatters = {}
-        
+
```

Comment:
While my editor (and I) also hate ZeP's inability to do whitespace properly, polluting commits with these whitespace changes make it harder to spot the real changes. Please try not to do it (or, if you do want to, do it in a commit on its own so it doesn't give me hideous rebase conflicts when I try and review your work six months late :P )